pkg/sources/fake/fake.go

```diff
@@ -14,122 +14,210 @@ const (
 	readInterval = 400 * time.Millisecond
 )
 
+// Enum for CAN message types, as defined in can_messages.asciipb and can_msg_defs.ts
```

Comment:
I think we can just use the `codegen-tooling` outputs here instead of duplicating this work? All these are currently being generated by `can_msg_defs.mako.go`

```golang
// For setting the CAN message ID
const (
	SystemCanMessageBpsHeartbeat           CanMsgId = 0
	SystemCanMessagePowerDistributionFault CanMsgId = 1
	SystemCanMessageBatteryRelayMain       CanMsgId = 2
	SystemCanMessageBatteryRelaySlave      CanMsgId = 3
```

Probably just merge @ckitagawa's change and then integrate it to our build system (or for the time being, just do a copy/paste to unblock yourself if you're struggling with that).

Reply:
Yeah, good idea. Calder mentioned doing this but I haven't figured out how to use it yet; I'll look into it.

Reply:
OK, I've added the file that codegen generates (with modifications to appease Golint) and I'm using that for the CAN message IDs.